functorch/_src/make_functional.py

```diff
@@ -310,7 +316,7 @@ def forward(self, params, *args, **kwargs):
             _swap_state(self.stateless_model, self.names_map, old_state)
 
 
-def make_functional(model: nn.Module):
+def make_functional(model: nn.Module, disable_autograd_tracking: bool = False):
     """make_functional(model) -> func, params
```

Comment:
nit: (sorry I missed this earlier) We should update this to match the new signature

Reply:
@samdow sorry for delay. Actually, I didn't change this part of docstring as `disable_autograd_tracking` arg is not essential and IMO `make_functional(model) -> func, params` is showing essential usage.
What do you think ? I can change it though if you confirm. Thanks

Reply:
Oh, I see, I haven't remarked that previously. Thanks! Let me update the docstring here as well